### PR TITLE
Paginate Log Group View

### PR DESCRIPTION
**Pull request by hunterwerlla**
- Paginate the Log Group view
- Break out loading on scroll into its own utils file
- Further genericize `LogStreamActor` into `LogActor`
- Prevent user column resize or sort
## Screenshots
Kapture 2020-04-10 at 12 47 39

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by kiiadi** [comment]
awesome feature!

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/LogActor.kt [hunk 1]
```diff
@@ -219,11 +224,13 @@ class LogStreamListActor(
...
-    logStream: String
+    private val logGroup: String,
+    private val logStream: String
 ) :
-    LogStreamActor(project, client, table, logGroup, logStream) {
+    LogActor<LogStreamEntry>(project, client, table) {
     override val emptyText = message("cloudwatch.logs.no_events")
+    override val tableErrorMessage = message("cloudwatch.logs.failed_to_load_stream", logStream)
+    override val notFoundText = message("cloudwatch.logs.log_stream_does_not_exist", logStream)
     override suspend fun loadInitial() {
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/LogActor.kt:234]: Add blank line above

Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/LogActor.kt:234]: Sure, added

**Pull request merged**